SparkPost/matchbox
# FE-947 Convert WindowEvent Component

Give your PR a recognizable title. For example: **FE-123: Ticket Title** or **Resolve Issue #123: Description**.

### What Changed
- Converts WindowEvents component to stateless functional component

### How To Test or Verify
- Start storybook `npm run start:storybook`
- Check that action is working in actions panel
- Make sure tests are valid and passing

### PR Checklist
